### PR TITLE
Modify CLI guide for docs

### PR DESCRIPTION
**Pull request by alphaX86**
**Notes for Reviewers**

This PR fixes 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e52da36647a46aae0b0278c2e1cdc370cb6b1d7a at: https://646857333e59cb5c064c334e--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f43c4cce7b78216d32bc1d9bdf279a92ffeb22ea at: https://646859c5e49d3260966a4c88--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-cli.md [hunk 1]
```diff
@@ -156,38 +156,40 @@ Use:   "start",
...
+{% endcapture %}
 {% include code.html code=code_content %}
-Certain symbols are used in the screenshot section, each having it's own function
-
-- `!` - Used to denote as comment and the line will be ignored by the script
-- `*` - Caption for the screenshot
-- `#` - Markdown code for adding the screenshot in the doc page
 
-**NOTE: It is advised not to modify the changes in `docs` folder, rather should be done in `mesheryctl` folder as the changes will get overwritten by the script.**
+**NOTE: It is advised not to modify the changes in `docs` folder, rather should be done in `mesheryctl` folder as the changes will get overwritten by the script. Also, the `index` page has to be manually updated by editing the `docs/_data/mesheryctlcommands/cmds.yml` file to appear the generated file in the command reference index table.**
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-cli.md:192]: Let's highlight these two instructions more prominently and as to separate notes. Please given them each short titles as well; e.g. **Adding New or Removing Existing Commands**

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/project/contributing/contributing-cli.md (hunk 1 above)
Comment (alphaX86) [reply docs/pages/project/contributing/contributing-cli.md:192]: Copy. Will get it done 👍

**Comment by l5io** [comment]
🚀 Preview for commit c9050e6b70fb9c1c6206218e43787b0ef04eea9f at: https://646b4157e6ec99127d7be2b0--meshery-docs-preview.netlify.app

**Review by abdullah1308 — APPROVED** [review]

**Comment by abdullah1308** [comment]
LGTM

**Comment by Chadha93** [comment]
@alphaX86  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Review by leecalcote — APPROVED** [review]

**Pull request merged**